workflow/pred_temperature_autoencoder.ipynb

```diff
@@ -0,0 +1,1298 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    target_series_sel = target_field_resample['ts'].sel(cluster=3)
</pre><p>This cluster won't be available from the data downloaded with era5cli</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI4S2S/cookbook/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Yes, we need to update this when we include downloading into our workflow. I will run each notebook again to make sure that everything still works.</p>